system/settings/settings.xml

```diff
@@ -1821,38 +1821,64 @@
     <category id="upnp" label="20187" help="36322">
       <requirement>HAS_UPNP</requirement>
       <group id="1" label="16000">
-        <setting id="services.upnpserver" type="boolean" label="21360" help="36323">
+        <setting id="services.upnp" type="boolean" label="1270">
           <level>0</level>
           <default>false</default>
           <control type="toggle" />
         </setting>
+        <setting id="services.upnpserver" type="boolean" label="21358" help="36323">
+          <level>1</level>
```

Comment:
The idea is to enable/disable UPnP altogether. As such, it makes sense to me that anything related to UPnP be under the new setting at level 0.